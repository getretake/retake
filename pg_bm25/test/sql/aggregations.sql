-- Bucket aggregation 
SELECT * FROM paradedb.aggregation('aggregations_bm25_index', '{"aggs": {histogram: {field: "rating", interval: 2}}}');

-- Metrics aggregation
SELECT * FROM paradedb.aggregation('aggregations_bm25_index', '{"aggs": {avg: {field: "rating"}}}');
