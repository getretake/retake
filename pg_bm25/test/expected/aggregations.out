-- Bucket aggregation 
SELECT * FROM paradedb.aggregation('idxaggregations', '{"aggs": {"histogram": {"field": "rating", "interval": 2}}}');
                                                     aggregation                                                     
---------------------------------------------------------------------------------------------------------------------
 {"aggs": {"buckets": [{"key": 0.0, "doc_count": 1}, {"key": 2.0, "doc_count": 12}, {"key": 4.0, "doc_count": 28}]}}
(1 row)

-- Metrics aggregation
SELECT * FROM paradedb.aggregation('idxaggregations', '{"aggs": {"avg": {"field": "rating"}}}');
               aggregation               
-----------------------------------------
 {"aggs": {"value": 3.8536585365853657}}
(1 row)

